Style Guide and Diataxis Review: Wallet I (#975)
iotaledger/wallet.rs: documentation/docs/libraries/overview.md

- node.js
- account
- multiple
- explanation 
---
# IOTA Wallet Libraries

> The `wallet.rs` library is primarily written in Rust but you can also find bindings written in Node.js, Python, and Java:

Reviewer: It is written in Rust only, not primarily. Only its bindings are in different languages (because they act like bridges to the rust library for programs written in other languages).

- [Rust](rust/getting_started.md).
- [Node.js](nodejs/getting_started.md).
- [Python](python/getting_started.md).
- [Java](java/getting_started.md).

## Getting Started

We recommended that you start your interactions with IOTA on the [devnet](https://wiki.iota.org/learn/networks/testnets#iota-20-decentralized-devnet). The _devnet_ will allow you to safely get acquainted with the `wallet.rs` library, without the risk of losing any funds if you make a mistake along the way. You can use this API load balancer: `api.lb-0.h.chrysalis-devnet.iota.cafe`.  

> We also have a network explorer that is available at [IOTA Tangle Explorer](https://explorer.iota.org/devnet). You can use the network explorer to view transactions and data stored in the IOTA Tangle.    

Reviewer 2: ```suggestion
You can also use the network explorer available at [IOTA Tangle Explorer](https://explorer.iota.org/devnet) to view transactions and data stored in the IOTA Tangle.    
```

> To properly test value-based transactions on the devnet, you are going to need some tokens! You can get some devnet tokens through our [faucet](https://faucet.chrysalis-devnet.iota.cafe/).

Reviewer 2: ```suggestion
To properly test value-based transactions on the devnet, you will need some tokens. You can get some devnet tokens using the [faucet](https://faucet.chrysalis-devnet.iota.cafe/).
```

## The Library in a Nutshell

> `wallet.rs` uses an account model so you can create an account for each of your users. You could also take another approach and use one account and generate many addresses, which you can link to your users in your database.

Reviewer 2: ```suggestion
`wallet.rs` uses an account model so you can create an account for each of your users. You can also take another approach and use one account and generate many addresses to link to your users in your database.
```

> Using the library gives you the ability to assign a meaningful alias to each account. Users may also segregate their funds across multiple accounts or multiple addresses. It is up to a developer whether he chooses a single-account approach or multi-account approach.

Reviewer 2: ```suggestion
Using the library allows you to assign a meaningful alias to each account. Users may also segregate their funds across multiple accounts or multiple addresses. It is up to a developer to choose a single-account approach or a multi-account approach.
```
